jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/MojoExceptionBuilder.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.maven;
+
+import org.apache.maven.plugin.MojoExecutionException;
+
+/** Builds a {@link MojoExecutionException} with a useful suggestion. */
+class MojoExceptionBuilder {
```

Comment:
This seems over-engineered. Perhaps just use a MojoException?

Reply:
Also not used in this PR.

Reply:
Oh this was removed in a prior PR - I must have somehow merged incorrectly and caused this to come back. Removed.